src/lib/bootloader/grub_install.rb

```diff
@@ -85,6 +86,9 @@ def report_failure(exception)
     # creates basic command for grub2 install without specifying any stage1
     # locations
     def basic_cmd(secure_boot, trusted_boot, update_nvram)
+      if secure_boot && (target == "powerpc-ieee1275") then
+        return ["/usr/bin/dd", "status=none", "if=/usr/share/grub2/powerpc-ieee1275/grub.elf"]
```

Comment:
so it means that grub2-install call is skipped completely? As you replace it with dd. And as I write in bug report I kind of expect that grub2 itself will do this dd, not all tools using grub